hs_model_program/static/js/model_program.js

```diff
@@ -0,0 +1,179 @@
+/**
+ * Created by tonycastronova on 10/12/15.
+ */
+
+$alert_error = '<div class="alert alert-danger" id="error-alert"> \
+                    <button type="button" class="close" data-dismiss="alert">x</button> \
+                    <strong>Error </strong> \
+                    failed to retrieve metadata for this resource.\
+                </div>'
```

Comment:
The function `metadata_update_ajax_submit()` changes this variable, and thus the error that will be used after it is called. Is that he behavior you really want?